net/redistest/redistest.go

```diff
@@ -10,6 +10,62 @@ import (
 	"github.com/testcontainers/testcontainers-go/wait"
 )
 
+func NewBenchRedis(b *testing.B) (address string, done func()) {
+	return NewBenchRedisWithPassword(b, "")
+}
+
+func NewBenchRedisWithPassword(b *testing.B, password string) (address string, done func()) {
```

Comment:
Could you check which methods of testing.B we need and maybe we can change existing constructors to use [common TB](https://pkg.go.dev/testing#TB) interface?